vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -604,7 +604,7 @@ testPamDSettings() {
 
   # We only want to run this test on Mariner 2.0
   # So if it's anything else, report that we're skipping the test and bail.
-  if [[ "${os_sku}" != "CBLMariner" || "${os_version}" != "2.0" ]]; then
+  if [[ "${os_sku}" != "CBLMariner" && "${os_sku}" != "Azurelinux" || "${os_version}" != "2.0" ]]; then
```

Comment:
Nit: We can clean up these `|| "${os_version}" != "2.0"` checks now that we're no longer building Mariner V1 images in AgentBaker.

Reply:
Fixed.